cirq/ops/parity_gates.py

```diff
@@ -20,7 +20,7 @@
 
 from cirq import protocols
 from cirq._compat import proper_repr
-from cirq.ops import gate_features, eigen_gate
+from cirq.ops import gate_features, eigen_gate, ZPowGate, CZPowGate
```

Comment:
Prefer importing the submodule that defines these.

Reply:
Done.